### PR TITLE
docs: Add instructions for running the app locally

### PR DESCRIPTION
**Pull request by nikiburggraf**
Based off of yesterday's work getting this to run on my machine. Please let me know if I should be more or less verbose on any step!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,49 @@ Deploying supabase via Kubernetes allows for a cloud agnostic deployment as long
...
+   1. Android SDK Platform
+   1. Android SDK Command-line tools
+   1. Android SDK Build-Tools
+   1. Android SDK Platform-Tools
+   1. Android Emulator
+3. Install Pixi: https://github.com/prefix-dev/pixi?tab=readme-ov-file#installation
+4. In the package's directory, run
+```
+pixi run install-tools
+```
```
Comment (lsetiawan) [inline README.md:42]:
```suggestion
4. In the package's directory, run
    
    ```console
    pixi run install-tools
    ```
```

Could you modify the formatting a little bit for these.

File: README.md [hunk 2]
```diff
@@ -24,6 +24,49 @@ Deploying supabase via Kubernetes allows for a cloud agnostic deployment as long
...
+5. Accept the Android licenses (or check that you have already done so) by running
+```
+pixi run flutter doctor --android-licenses
```
Comment (lsetiawan) [inline README.md:45]:
```suggestion
pixi run check-flutter --android-licenses
```

File: README.md [hunk 3]
```diff
@@ -24,6 +24,49 @@ Deploying supabase via Kubernetes allows for a cloud agnostic deployment as long
...
+```
+6. Check that you're ready to run Flutter by running the following (there may be an issue indicated with XCode -- that is okay and can be ignored)
+```
+pixi run flutter doctor
```
Comment (lsetiawan) [inline README.md:49]:
```suggestion
pixi run check-flutter
```

File: README.md [hunk 4]
```diff
@@ -24,6 +24,49 @@ Deploying supabase via Kubernetes allows for a cloud agnostic deployment as long
...
+7. Set environment variables with your Docker username and password
+```
+export DOCKER_USERNAME=<your docker username>
+export DOCKER_PASSWORD=<your docker password>
+```
+8. Run the Docker daemon
+9. Install the required submodules
+```
+git submodule update --init
+```
```
Comment (lsetiawan) [inline README.md:60]: Just added this to the pixi task. No need to list here.

File: README.md (hunk 1 above)
Comment (nikiburggraf) [reply README.md:42]: done :)

File: README.md (hunk 3 above)
Comment (nikiburggraf) [reply README.md:49]: done

File: README.md (hunk 4 above)
Comment (nikiburggraf) [reply README.md:60]: will remove

**Pull request merged**